### DISCUSSION TITLE
Disable clustering

### DISCUSSION THREAD

**Discussion by Techbrunch** [projectdiscovery/nuclei#2062] [General]
Hello,

Is there a way to disable clustering of templates ?

I'm running nuclei using:

```
nuclei -u https://target.com -proxy http://127.0.0.1:8080 -vv
```

but I don't see the request for template `laravel-debug-enabled` in Burp. I suspect this might be because of clustering.

**Comment by geeknik** [comment]
Clustering works by nuclei making 1 HTTP request and then running ALL of the clustered templates against the response from that HTTP request. Which means you won't see singular HTTP requests for every template and that's okay. If you want to see the HTTP request for that particular template, you'd have to run it alone without other templates in the command line. Clustering saves time, saves bandwidth and limits the amount of noise in your target's log files. 👍🏻

**Reply by Techbrunch** [reply]
Hi geeknik, thanks for the info but this does not answer my question / suggestion. Ideally I think clustering should be an option. Also there's clearly something wrong happenning since no request is done to the specific endpoint:

```
/_ignition/health-check
```


https://github.com/projectdiscovery/nuclei-templates/blob/38c9989314b017514362df5bbcede9867c4e7443/misconfiguration/laravel-debug-enabled.yaml

**Reply by Techbrunch** [reply]
I think I found the issue, my target was protected by some kind of WAF which resulted in many errors but Nuclei is silently stopping when there are too many errors. I'll open a new issue to ask for no "silent" errors.

**Reply by Techbrunch** [reply]
The created issue: https://github.com/projectdiscovery/nuclei/issues/2065
